### DISCUSSION TITLE
background-color is different in design system and the home-office-kit

### DISCUSSION THREAD

**Discussion by SparkyUX** [UKHomeOffice/design-system#531] [General]
The Home Office [Design System colour page](https://design.homeoffice.gov.uk/styles/colour) states that the background-colour should be #F1F1F1 but when the home-office-kit is installed it sets the body background-colour to #F5F5F5 (home-office-kit/sass/_page.scss).
Which is the correct colour to use?

**Comment by tomyems** [comment]
I've had a look through the history and can't quite work out which one came first, I believe #F5F5F5 should be used, as the contrast work we've done has used the actual page background colour rather then the one on the colour page. Will do a bit more investigation and see if we should update the colour page to #F5F5F5

**Comment by tomyems** [comment]
Preview: https://66471f61d8a3d093c86cc0c8--hods.netlify.app/ of update to the background colour description on the colour page.

Implemented by #579
